Entity/CategoryManager.php

```diff
@@ -136,6 +176,31 @@ public function getRootCategories($loadChildren = true)
     }
 
     /**
+     * @param bool|true $loadChildren
+     *
+     * @return array
+     */
+    public function getRootCategoriesSplitByContexts($loadChildren = true)
```

Comment:
Should we add this (and the other new public methods) to the `CategoryManagerInterface` with the next major release?